server/src/main/java/com/hedera/block/server/BlockStreamService.java

```diff
@@ -0,0 +1,40 @@
+package com.hedera.block.server;
+
+import com.google.protobuf.Descriptors;
+import com.hedera.block.protos.BlockStreamServiceGrpcProto;
+import io.grpc.stub.StreamObserver;
+import io.helidon.webserver.grpc.GrpcService;
+
+import java.util.logging.Logger;
+
+import static io.helidon.webserver.grpc.ResponseHelper.complete;
+
+
+public class BlockStreamService implements GrpcService {
```

Comment:
I think Matt already implemented BlockStream Service, should this be HistoricBlockService?